backend/shared/shared/audit_log/README.md

```diff
@@ -2,70 +2,32 @@
 
 An audit logging package that can be used with `djangorestframework`.
 
-This audit logger allows you to save log messages from Django Rest Famework CRUD events. The serializer works by defining a DRF model viewset that inherits `AuditLoggingModelViewSet`.
+This audit logger allows you to save log messages from Django Rest Famework CRUD events to database. The serializer works by defining a DRF model viewset that inherits `AuditLoggingModelViewSet`.
 
 To start using this package, follow these steps:
 
-1. Add `djangorestframework` to requirements and add these settings:
+1. Add `djangorestframework` to requirements and add this setting:
     ```
+    env = environ.Env(
+        ...,
+        AUDIT_LOG_ORIGIN=(str, ""),
+    )
+
     # Audit logging
     AUDIT_LOG_ORIGIN = env.str("AUDIT_LOG_ORIGIN")
-    AUDIT_LOG_FILENAME = env.str("AUDIT_LOG_FILENAME")
 
-    if AUDIT_LOG_FILENAME:
-        if "X" in AUDIT_LOG_FILENAME:
-            import random
-            import re
-            import string
-    
-            system_random = random.SystemRandom()
-            char_pool = string.ascii_lowercase + string.digits
-            AUDIT_LOG_FILENAME = re.sub(
-                "X", lambda x: system_random.choice(char_pool), AUDIT_LOG_FILENAME
-            )
-        _audit_log_handler = {
-            "class": "logging.handlers.RotatingFileHandler",
-            "filename": AUDIT_LOG_FILENAME,
-            "maxBytes": 100_000_000,
-            "backupCount": 1,
-            "delay": True,
-        }
-    else:
-        _audit_log_handler = {
-            "class": "logging.StreamHandler",
-            "formatter": "verbose",
-        }
-    
-    
-    LOGGING = {
-        ...
-        "formatters": {
-            "verbose": {
-                "format": "%(asctime)s p%(process)d %(name)s %(levelname)s: %(message)s",
-            }
-        },
-        "handlers": {
-            ...
-            "audit": _audit_log_handler,
-        },
-        "loggers": {
-            ...
-            "audit": {
-                "level": "INFO",  # Audit log only writes at INFO level
-                "handlers": [
-                    "audit",
-                ],
-            },
-        },
-    }
+    INSTALLED_APPS = [
+        ...,
+        "shared.audit_log",
+    ]
     ```
 
-2. Add `AUDIT_LOG_ORIGIN` and `AUDIT_LOG_FILENAME` to env variables
-
+2. Add `AUDIT_LOG_ORIGIN` to env variables.
 
```

Comment:
Maybe add instructions how to produce log events if not using AuditLoggingModelViewSet, like from a background cron job / django management command / other?